CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+0.27.0 (2020-12-04)
+-------------------
+
+- Bumping `snowflake-connector-python` across all componenets that uses to `2.3.6`
+- Tagging all queries issues to Snowflake by FastSync Snowflake and singer target-snowflake.
+- Add ssl support to mongodump in FastSync mongodb.
+- Fix issues build PPW docker images
+- Update doc.
+
+
```

Comment:
can we add tap-mysql spatial data type support?

Reply:
Oh, missed those! Will do, thanks

Reply:
Done in https://github.com/transferwise/pipelinewise/pull/585/commits/bbf49ced5cff9a1f812a57cf2e80186941faf2f7